game-core/src/main/java/games/strategy/engine/framework/startup/ui/ClientSetupPanel.java

```diff
@@ -254,6 +254,10 @@ public IChatPanel getChatPanel() {
     actions.add(clientModel.getHostBotChangeToAutosaveClientAction(this, AUTOSAVE_TYPE.AUTOSAVE));
     actions.add(clientModel.getHostBotChangeToAutosaveClientAction(this, AUTOSAVE_TYPE.AUTOSAVE_ODD));
     actions.add(clientModel.getHostBotChangeToAutosaveClientAction(this, AUTOSAVE_TYPE.AUTOSAVE_EVEN));
+    actions.add(clientModel.getHostBotChangeToAutosaveClientAction(this, AUTOSAVE_TYPE.AUTOSAVE_END_TURN));
+    actions.add(clientModel.getHostBotChangeToAutosaveClientAction(this, AUTOSAVE_TYPE.AUTOSAVE_BEFORE_BATTLE));
+    actions.add(clientModel.getHostBotChangeToAutosaveClientAction(this, AUTOSAVE_TYPE.AUTOSAVE_AFTER_COMBAT_MOVE));
```

Comment:
Should there be an action for `AUTOSAVE_AFTER_BATTLE` here?  Just wondering why the difference between this code and the corresponding actions in `GameSelectorPanel`.

**EDIT:** Does this have something to do with the comment in `ServerGame#autoSaveAfter()` that states it will only ever be called for a headed game?